test/ts-scenario/features/discovery.feature

```diff
@@ -72,4 +72,10 @@ Feature: Configure Fabric using CLI and submit/evaluate using a network Gateway
 
 	Scenario: Using a Gateway I can use transient data
 		When I use the discovery gateway named myDiscoveryGateway to submit a transaction a 100 times with args [createCar,2001,Ford,F350,red,Sam] for contract fabcar instantiated on channel discoverychannel
-		Then The gateway named myDiscoveryGateway has a submit type response
\ No newline at end of file
+		Then The gateway named myDiscoveryGateway has a submit type response
+
+	Scenario: Using a Gateway I can use system chaincodes
```

Comment:
This might be better as two scenarios. Then if a scenario fails we know immediately whether it was the submit or evaluate that didn't work. It will also ensure both cases are tested rather than skipping the evaluate test if the submit fails.